pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>com.sinch.sdk</groupId>
     <artifactId>sinch-sdk-java</artifactId>
-    <version>1.2.0-SNAPSHOT</version>
+    <version>1.2.1-SNAPSHOT</version>
```

Comment:
Should go to 1.3-next

Reply:
Plan was to follow these steps:
 - release it onto a 1.2.1 version for a dedicate Numbers improvement
 - then add it to a not yet planned 1.3-next